cnvlib/reference.py

```diff
@@ -44,14 +45,15 @@ def bed2probes(bed_fname):
     table["gene"] = regions.data["gene"] if "gene" in regions.data else "-"
     table["log2"] = 0.0
     table["spread"] = 0.0
-    return CNA(table, {"sample_id": core.fbase(bed_fname)})
+    return CNA(table, {"sample_id": core.fbase(bed_fname)}) # todo: find all invocations of CNA() and forward param
```

Comment:
What does this mean? I don't see a new parameter used here.

Reply:
Maybe the wording here is unfortunate. I wanted to make sure a decision is taken, not only for `read_cna` but also when invoking  the constructor (see my other comment).